Model/Api/CartManagement.php

```diff
@@ -24,6 +24,8 @@
 use Magento\Store\Model\StoreManagerInterface;
 use Bolt\Boltpay\Helper\Bugsnag;
 use Bolt\Boltpay\Helper\Cart as CartHelper;
+use Magento\Quote\Model\MaskedQuoteIdToQuoteIdInterface;
+use Bolt\Boltpay\Model\Api\Data\GetCartIdByMaskedQuoteIdData;
```

Comment:
please refactor code after removing the new "GetCartIdByMaskedQuoteIdData" class